euclidean_gcd/euclidean_gcd.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import "fmt"
+
+// EuclideanGCDIterative finds GCD of given numbers by iterative method
```

Comment:
It would be great if you explain the information about the parameters and show the time complexity of the function as well.